daemon/src/taker_inc_message_actor.rs

```diff
@@ -1,12 +1,11 @@
 use crate::model::cfd::Origin;
-use crate::taker_cfd_actor::TakerCfdActor;
-use crate::{taker_cfd_actor, wire};
```

Comment:
What I specifically like about this pattern of referring to types via their parent module is that it results in relatively clean and short imports!

Reply:
yes, and indeed not losing it can leave out important context when reading a line - ie. which actor is it ;D

It would be cool to lint that someday indeed.